script

```diff
@@ -689,6 +704,12 @@ only_send_error_notifications="0"
               # run compare function. compare will not run if compare_files is disabled.
               run_compare "$disk" "$backup_location/$vm/$new_disk" "vdisk"
 
+            elif [ "$inline_zstd_compress" -eq 1 ]; then
```

Comment:
I'll have to investigate that a bit more.  If we want zstd compression support without timestamps it might require a slightly different approach still.

Reply:
A new approach has been committed that should address this.

Reply:
A new approach has been committed that should address this.